test/test_support.ts

```diff
@@ -299,17 +300,71 @@ export function getSourceMapWithName(
   return sourceMapTextToConsumer(findFileContentsByName(filename, files));
 }
 
+function removed(str: string) {
+  return '\x1B[37;41m' + str + '\x1B[0m';
+}
+
+function added(str: string) {
+  return '\x1B[37;32m' + str + '\x1B[0m';
+}
+
+/**
+ * A Jasmine "compare" function that compares the strings actual vs expected, and produces a human
+ * readable, colored diff using diff-match-patch.
+ */
+function diffStrings(actual: {}, expected: {}) {
+  if (actual === expected) return {pass: true};
+  if (typeof actual !== 'string' || typeof expected !== 'string') {
+    return {pass: false, message: `toEqualWithDiff takes two strings, got ${actual}, ${expected}`};
+  }
+  const dmp = new DiffMatchPatch();
+  dmp.Match_Distance = 0;
+  dmp.Match_Threshold = 0;
+  const diff = dmp.diff_main(expected, actual);
+  dmp.diff_cleanupSemantic(diff);
+  if (!diff.length) return {pass: true};
+  let message = '\x1B[0;0mstrings differ:\n';
+  for (const d of diff) {
+    switch (d[0]) {
+      case DIFF_EQUAL:
+        message += d[1];
+        break;
+      case DIFF_DELETE:
+        // light gray on red.
+        message += '\x1B[37;41m' + d[1] + '\x1B[0;0m';
```

Comment:
Super minor, but:

`\x1b[0;0m` can just be written `\x1b[0m`, the 0 resets everything. Here and below.

Reply:
Done.